features/reports_by_day_metrics.feature

```diff
@@ -0,0 +1,207 @@
+@api @metrics
+Feature: Reports by day metrics
+
+  Users should be able to retrieve the number of reports by day of an organization.
+  The users can filter the by project and/or by user.
+  Finally, the users can retrieve between 1 and 120 days of data.
+
+  The following result is provided:
+  - runsCount for each day
+  - date for each day
+
+
+  Background:
+    Given private organization Rebel Alliance exists
+    And user hsolo who is a member of Rebel Alliance exists
+    And user lskywalker who is a member of Rebel Alliance exists
+    And project X-Wing exists within organization Rebel Alliance
+    And test result report report 1 was generated 2 days ago for organization Rebel Alliance
```

Comment:
"test result report report 1" reads very weird. I would prefer replacing "report 1" by "R1" and similarly for other such occurrences.